theforeman.org/pipelines/release/source/foreman.groovy

```diff
@@ -37,8 +37,10 @@ pipeline {
                     }
                     post {
                         always {
-                            cleanup(env.RUBY_VER, env.GEMSET)
                             junit(testResults: 'jenkins/reports/unit/*.xml')
+                        }
+                        cleanup {
+                            cleanup(env.RUBY_VER, env.GEMSET)
```

Comment:
This looks confusing. Perhaps our `cleanup` should really be `rvmCleanup` or similar. However, that's for another day.